### PR TITLE
Make SDK model agnostic

### LINKED ISSUE #80: Make the SDK drone model agnostic
Blueye now sells two different drone models:

- Blueye Pioneer
- Blueye Pro

The SDK is compatible with both models already, which should be communicated in the documentation.

As for the main `Pioneer` class, I am not sure if we should rename it to something generic such as `BlueyeDrone`. The SDK is not officially released yet and in terms of backward compatibility, we could either add an empty `Pioneer` class that derives from `BlueyeDrone` or we could accept a breaking change?

```python
class Pioneer(BlueyeDrone):
  pass
```

This way we don't break existing code, such as our own test scripts, or customers who have started using it.

### PR DESCRIPTION
**Pull request by follesoe**
Ensure we use generic phrases such as Blueye Drones, or mention both Pro and Pioneer.

**Description**
As the Blueye Pro is released the SDK should be updated to reflect that it is compatible with both Blueye Pioneer and Blueye Pro.

Fixes #80

### Todo
- [x] Decide on new class name ~~, leaning towards `Drone` now, but open for suggestions~~ Going for `Drone`.
- [x] Update variable names in tests 
- [x] Update documentation text
- [x] Update documentation examples
- [x] Update docstrings

#### Checklist before merging

- [x] Run the tests while connected to a drone

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,23 +12,23 @@ _________________
...
-is connected via WiFi to a surface unit, and the Pioneer is connected to the surface unit via a tether cable.
+## This SDK and the Blueye drones
+A Blueye drone is normally controlled via a mobile device through the Blueye App ([iOS](https://apps.apple.com/no/app/blueye/id1369714041)/[Android](https://play.google.com/store/apps/details?id=no.blueye.blueyeapp)). The mobile device
+is connected via Wi-Fi to a surface unit, and the Pioneer or Pro is connected to the surface unit via a tether cable.
 
-This python SDK exposes the functionality of the Blueye app through a Python object. The SDK enables remote control of the Pioneer as well as reading telemetry data and viewing video streams, it is not meant for executing code on the Pioneer.
-To control the Pioneer you connect your laptop to the surface unit WiFi and run code that interfaces with the Pioneer through the Pioneer Python object.
+This python SDK exposes the functionality of the Blueye app through a Python object. The SDK enables remote control of the Blueye Pioneer and Blueye Pro as well as reading telemetry data and viewing video streams. It is not meant for executing code on the Pioneer.
```
Comment (aksell) [inline README.md:30]:
```suggestion
This python SDK exposes the functionality of the Blueye app through a Python object called Pioneer. The SDK enables remote control of the Blueye Pioneer and Blueye Pro as well as reading telemetry data and viewing video streams. It is not meant for executing code on the Pioneer.
```
If we don't want to introduce a separate Pro object or rename the Pioneer class to something model agnostic. We could perhaps mention the naming quirk of the Pioneer class handling both the Pro and the Pioneer in a note here like this 👇 

**Note:** The python object Pioneer will allow you to interface with both the Pioneer and the Pro drone. This is a quirk of the Pro drone being introduced later then the Pioneer.

Comment (sindrehan) [reply README.md:30]: I think the best option is to rename the Pioneer class and deprecate it before the 1.0 release.

**Comment by sindrehan** [comment]
FYI: I'm rebasing this onto master to sort out the merge conflicts.

**Review by aksell — APPROVED** [review]
LGTM, nice work :smile:

File: README.md [hunk 2]
```diff
@@ -12,23 +12,23 @@ _________________
...
 ![SDK demo](https://user-images.githubusercontent.com/8504604/66751230-d05c7e00-ee8e-11e9-91cb-d46b433aafa5.gif)
 
-## About The Pioneer
-The Blueye Pioneer is a underwater drone designed for inspections. It is produced and sold by the Norwegian company [`Blueye Robotics`](https://www.blueyerobotics.com/).
+## About Blueye Underwater Drones
+The Blueye Pioneer and Blueye Pro are underwater drones designed for inspections. It is produced and sold by the Norwegian company [`Blueye Robotics`](https://www.blueyerobotics.com/).
 Here is a Youtube video  that gives a overview of the system and its specifications.
 
-[![about the Pioneer video](https://img.youtube.com/vi/_-AEtr6xOP8/0.jpg)](https://www.youtube.com/watch?v=_-AEtr6xOP8)
+[![about the Blueye Pro video](https://images.ctfassets.net/qpo1z0ycuc50/2JLqOXRoSIMe370zfrtgQV/445995589cbdb4e6baa406d97bcbf667/DSC05337_3_Mathilde_edit.jpg?w=1920&q=80)](https://www.youtube.com/watch?v=HI2CQbnCsoU)
```
Comment (sindrehan) [inline README.md:24]: I think we should add have a play button on top of the image to highlight that it links to a youtube video. I've created one with a [random website](http://addplaybuttontoimage.way4info.net/), but we might want to use a nicer one instead.
```suggestion
[![about the Blueye Pro video](https://user-images.githubusercontent.com/8504604/94565079-05f9a300-0269-11eb-85a8-b2f58bcdcdc2.png)](https://www.youtube.com/watch?v=HI2CQbnCsoU)
```

**Pull request merged**